### PR TITLE
added Gabriele Paciocco personal details to data

### PR DESCRIPTION
**Pull request by GabePaciocco**
## Summary of changes

This PR was made to add Gabriele Paciocco's Personal information to the group website. This change will add my picture, name, as well as contact information for refernece. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -261,19 +261,16 @@
...
+  description: 'Gabriele is an NPRE undergraduate at UIUC. His interests in Math and Science motivate him to study the future of nuclear reactor devlopement. 
+    - url: mailto:Agp9@illinois.edu
       icon: envelope-o
       name: Email
-    - url: https://www.google.com/maps/d/edit?mid=zwJ5Sc8HIDN4.k3fYHsg-biWE&usp=sharing
-      icon: map-marker
-      name: 118 Talbot Lab
+    - url: https://www.linkedin.com/in/gabriele-paciocco-b311b41a1/
+      icon: linkedin
+      name: Linkedin
```
Comment (nsryan2) [inline _data/people.yml:276]:
```suggestion
- name: You
  pic: /img/people/you.png
  dates: now
  role: Group Member
  src: /positions
  url: /positions
  description: 'Ideal ARFC group members seek to advance nuclear energy challenges through advanced computing, physics-informed modeling, and sophisticated simulation'
  social:
    - url: mailto:katyhuff@gmail.com
      icon: envelope-o
      name: Email
    - url: https://www.google.com/maps/d/edit?mid=zwJ5Sc8HIDN4.k3fYHsg-biWE&usp=sharing
      icon: map-marker
      name: 118 Talbot Lab
```

**Comment by yardasol** [comment]
Hi Gabriele!
Thanks for adding yourself to the website. I look forward to working with you in near future.

I like the level of detail in your PR description. This is about the level of detail we expect for PRs (Obviously the bigger your PR, the more details you will have to give)

Don't forget to mark the checkbox for "New feature" under "Types of Changes". Also, if you've run the website on your local machine and it works, you should check the "All new and existing tests pass" box.

Make sure to take a look at the comment I left regarding adding your photo!

- [x] Read the PR description
- [x] Read through all the changes, considering the following questions.
  - [x] Is there anything you can praise about this PR? Start with
        praise.
  - [x] Is the code clear and clean? (see Robert C. Martin's
        [Clean Code](https://i-share-uiu.primo.exlibrisgroup.com/discovery/fulldisplay?docid=alma99944155312205899&context=L&vid=01CARLI_UIU:CARLI_UIU&tab=LibraryCatalog&lang=en))
  - [x] Is there enough documentation?
  - [x] Does the programming style meet the requirements of the
        repository ([PEP8](https://www.python.org/dev/peps/pep-0008/) for python,
        [google C++ style guide](https://google.github.io/styleguide/cppguide.html), etc.)
  - [ ] If there is no continuous integration, check out the branch
        locally, build, and run the tests.
  - [ ] Do the tests pass on your machine?
  - [x] Is the PR free of random cruft (built files, `.swp` files,
        etc.)?
  - [x] Do all files in the PR belong in the repository?
  - [x] If the PR adds files or data, are these new files compatible
        with the repository license?
  - [x] Make a review, leaving kind comments and suggesting changes
        where needed (to resolve the above).
  - [ ] Has the author resolved all of the comments and suggestions
        in your review?
- [ ] When you approve of the PR, merge and close it.
- [x] Thank the author for their contribution.

**Comment by nsryan2** [comment]
Good points from Olek! I would also add that your PR should include the picture of you as well. It seems to have been lost in the shuffle, but it should be added in the same location as you're directing it to in the `pic:` line

**Comment by GabePaciocco** [comment]
@nsryan2 @yardasol I think I have resolved your comments let me know if there are any further edits needed.

**Comment by munkm** [comment]
@nsryan2 @yardasol are your review comments resolved now?

**Comment by nsryan2** [comment]
It all looks good to me, sorry @GabePaciocco I totally forgot about this. It looks like you've addressed @yardasol's comments too, but I'll let him confirm that!

**Review by yardasol — COMMENTED** [review]
Looks good to me! Sorry for the long wait on this y'all.

**Pull request merged**